pkg/services/authn/authnimpl/service.go

```diff
@@ -152,6 +152,8 @@ func ProvideService(
 		s.RegisterPostAuthHook(sync.ProvideOAuthTokenSync(oauthTokenService, sessionService).SyncOauthTokenHook, 60)
 	}
 
+	s.RegisterPostAuthHook(sync.ProvidePermissionsSync(accessControlService).SyncPermissionsHook, 70)
```

Comment:
I think we need to run the hook after FetchSyncedUserHook to make sure teams are loaded correctly

Reply:
Confirmed, Teams do need to be loaded

Reply:
(priority 110?)